### PR TITLE
Migrere til gcp

### PR DESCRIPTION
**Pull request by JulieHillRoa**

### CODE REVIEW

File: .nais/application/application-config-dev-gcp.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    cpuThresholdPercentage: 75
+  resources:
+    limits:
+      cpu: "2"
+      memory: 2048Mi
+    requests:
+      cpu: 500m
+      memory: 1024Mi
+  ingresses:
+    - https://veilarbveileder.intern.dev.nav.no
```
Comment (dragutin-nav) [inline .nais/application/application-config-dev-gcp.yaml:35]: ingress should not be the same in fss and gcp (I think)

File: .nais/application/application-config-prod-gcp.yaml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    requests:
+      cpu: 500m
+      memory: 1024Mi
+  ingresses:
+    - https://veilarbveileder.intern.nav.no
+  vault:
+    enabled: true
+    paths:
+      - kvPath: /kv/prod/fss/veilarbveileder/default
+        mountPath: /var/run/secrets/nais.io/vault
```
Comment (dragutin-nav) [inline .nais/application/application-config-prod-gcp.yaml:40]:
```suggestion
```

File: .nais/application/application-config-prod-gcp.yaml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    cpuThresholdPercentage: 75
+  resources:
+    limits:
+      cpu: "2"
+      memory: 2048Mi
+    requests:
+      cpu: 500m
+      memory: 1024Mi
+  ingresses:
+    - https://veilarbveileder.intern.nav.no
```
Comment (dragutin-nav) [inline .nais/application/application-config-prod-gcp.yaml:35]: ingress should not be the same in fss and gcp (I think)

File: .nais/application/application-config-prod-gcp.yaml (hunk 2 above)
Comment (JulieHillRoa) [reply .nais/application/application-config-prod-gcp.yaml]: Done :D

File: .nais/application/application-config-prod-gcp.yaml (hunk 3 above)
Comment (JulieHillRoa) [reply .nais/application/application-config-prod-gcp.yaml]: done :D

**Review by dragutin-nav — APPROVED** [review]

**Pull request merged**